open-company/open-company-web
# Don't force login

card: https://trello.com/c/xptRIfLA

test:
- remove the JWT token from your cookies
- check some companies data to be sure that everything is working (should be read only since you don't have a token)
- do you see the login button at the top right corner? good (asset is needed for the button)
- click the login button when you are on a company page
- complete the login flow
- do you land back on the same company page? good
- can you edit the company data? (should be if you are part of the same organization)
- merge
